### DISCUSSION TITLE
Can't see the workflow trace

### DISCUSSION THREAD

**Discussion by eric-easyhealth** [traceloop/openllmetry#1804] [Q&A] [answered]
I'm using Haystack2 with latest trackloop 0.26.4, followed the offical guidance to init the traceloop by following code:
```python
Traceloop.init(app_name="app_name", api_key="apikey")
``` 
it works perfect and shown me the openai track as following, but how can i enable the workflow track like what is suggested in the quick start document? 
PS: i'm using a pipeline in a python class, dummy code as below:
```python
from haystack import Pipeline
from haystack.components.generators.chat import OpenAIChatGenerator

class ChatPipelineViaRAG:
    def __init__(self, api_key:str = '')
        self.retrieve_pipeline = Pipeline()
        self.generator = OpenAIChatGenerator()
        ......
    
    def run(self, messages: List[ChatMessage]) -> dict[str, Any]:
        retrieved_documents = self.retrieve_pipeline.run(parameters...)
        self.messages = process_documents(messages, retrieved_documents)
        return self.generator.run(self.messages)
        

``` 
<img width="1524" alt="image" src="https://github.com/user-attachments/assets/15810338-9290-4e41-a3e8-6a627ce36ca3">
<img width="716" alt="image" src="https://github.com/user-attachments/assets/df1fc8c7-f8d8-4d8b-a28d-9800e8990d8a">

**Comment by nirga** [comment, marked as answer]
Thanks @eric-easyhealth! that seems to be a bug with Haystack, do you mind converting this to an issue in the repo? We'll take a look into this

**Reply by eric-easyhealth** [reply]
thanks for the confirmation, I created an issue and make this discussion as done.
